mlos_bench/README.md

```diff
@@ -47,152 +54,118 @@ With a [JSON5](https://spec.json5.org) [config file](./mlos_bench/config/) and c
-
-    If necessary, login to Azure and set your default subscription:
-
-    ```powershell
-    # If using az cli for the first time, a login will be required:
-    az login
-    # Make sure that GSL subscription is your default:
-    az account set --subscription "..."
-    ```
+From here onwards we assume we are in the project root directory.
+Ensure you have a conda environment (`mlos`) set up for executing `mlos_bench`.
+Create and activate the environment with:
 
-2. Generate access tokens to interact with Azure resources.
+```sh
+conda env create -f conda-envs/mlos.yml
+conda activate mlos
+```
 
-    A script at `./scripts/generate-azure-credentials-config.ps1` produces a JSON with Azure credentials.
-    This data is in the format that can be used by our framework.
+### 2. Make sure that you have Azure CLI tool installed and working
 
-    ```powershell
-    # If using for the az cli first time, a login will be required
-    ./scripts/generate-azure-credentials-config.ps1 > ./global_config.json
-    ```
+> Installation instructions for `az` (Azure CLI) [can be found here](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli).
 
-    On Linux, use `./scripts/generate-azure-credentials-config.sh` (requires `az` and `jq` to be installed).
+If necessary, login to Azure and set your default subscription:
 
-3. Copy the sample config tree from `mlos_bench/config` to another folder of your choice, e.g.,
+```powershell
+# If using az cli for the first time, a login will be required:
+az login
+# Make sure that GSL subscription is your default:
+az account set --subscription "..."
+```
 
-    ```sh
-    cp -R mlos_bench/config ./my-config
-    ```
+### 3. Generate access tokens to interact with Azure resources
 
-    > Note: there are several other examples in [`mlos_bench/config`](./mlos_bench/config/) that you can use as a starting point for your own experiments.
+A script at `./scripts/generate-azure-credentials-config.ps1` produces a JSON with Azure credentials.
+This data is in the format that can be used by our framework.
 
-4. Modify the service configuration files at `my-config/azure/service-*.jsonc` with your Azure setup data.
-E.g., in `my-config/azure/service-provision-vm.jsonc`, put your target resource group information, as well as desired VM name, denoted in `{{ ... }}` below.
+```powershell
+# If using for the az cli first time, a login will be required
+./scripts/generate-azure-credentials-config.ps1 > ./global_config_azure.json
+```
 
-    > All resource names have to either:
-    >
-    > - Not conflict with existing resource names in the resource group for a new deployment, or
-    > - Correspond to existing resources from a previous successful deployment.
-    >
-    > Re-using resources from a partial, unsuccessful deployment will likely fail. It is recommended to delete resources from those partial deployments.
+On Linux, use `./scripts/generate-azure-credentials-config.sh` (requires `az` and `jq` to be installed).
```

Comment:
```suggestion
On Linux (or inside the devcontainer), use the following:
```sh
./scripts/generate-azure-credentials-config.sh > ./global_config_azure.json
\`\`\` # FIXME: Remove backslash escapes in actual code - can't see to do that in the web editor

> Note: Requires `jq` to also be installed (comes preinstalled in the devcontainer).

```